### DISCUSSION TITLE
Custom domain for databases through one-click deployment

### DISCUSSION THREAD

**Discussion by SpuriousGer** [coollabsio/coolify#2353] [Q&A]
I might have overlooked this, but how can I set a custom domain for the PostgresSQL database? Is this possible? For easier migration later, I would love to set a certain domain.

**Comment by RustyDoggoBytes** [comment]
Have you a solution for this?

**Reply by flawnn** [reply]
> Have you a solution for this?

Own Docker Compose deployment with raw compose enabled - then set your own proxy settings

**Comment by SpuriousGer** [comment]
No, haven't heard anything and I can't set anything in the frontend.
